### PR TITLE
pkg/driver: native random fix for micro-ecc

### PR DESCRIPTION
**Pull request by d00616**
This should fix https://github.com/RIOT-OS/RIOT/issues/4621

Original micro-ecc random function accessing /dev/urandom. More to read about random/urandom: http://www.2uo.de/myths-about-urandom/

### CODE REVIEW

File: cpu/native/periph/random.c [hunk 1]
```diff
@@ -137,9 +137,12 @@ void _native_rng_init_hq(void)
 {
     DEBUG("_native_rng_init_hq\n");
     _native_syscall_enter();
-    dev_random = real_open("/dev/random", O_RDONLY);
+    dev_random = real_open("/dev/urandom", O_RDONLY | O_CLOEXEC);
     if (dev_random == -1) {
-        err(EXIT_FAILURE, "_native_rng_init_hq: open(/dev/random)");
+        dev_random = real_open("/dev/random", O_RDONLY | O_CLOEXEC);
+        if (dev_random == -1) {
+            err(EXIT_FAILURE, "_native_rng_init_hq: open(/dev/urandom|/dev/random)"");
```
Comment (cgundogan) [inline cpu/native/periph/random.c]: please remove the extra `"` at the end

**Comment by cgundogan** [comment]
Please address my comment - otherwise ACK

**Comment by d00616** [comment]
I had make a mistake. There are sqased to much. I open a new PR.

**Comment by Kijewski** [comment]
If you accidentally "lost" a revision and you can't remember the commit-hash, then you can still find the old head with `git reflog`. Just hard reset to this commit hash and the mistake will be undone.

**Comment by d00616** [comment]
> If you accidentally "lost" a revision and you can't remember the
> commit-hash, then you can still find the old head with |git reflog|.
> Just hard reset to this commit hash and the mistake will be undone.
>
>
Thanks. This command looks like its possible to undo my mistake next time.

**Pull request closed without merging**